oak-search-elastic/pom.xml

```diff
@@ -33,7 +33,8 @@
   <description>Oak Elasticsearch integration subproject</description>
 
   <properties>
-    <elasticsearch.version>7.17.7</elasticsearch.version>
+    <elasticsearch.hlrc.version>7.17.9</elasticsearch.hlrc.version>
+    <elasticsearch.java.version>8.5.3</elasticsearch.java.version>
```

Comment:
Any reason for not using the latest version at this time, 8.7.0? It should be compatible with any 8.X server. 
And maybe more importantly, 8.7.0 has a bulk ingester helper that at a first glance, seems to be what we need to drop the 7.17.X client. 
https://github.com/elastic/elasticsearch-java/pull/474
https://github.com/elastic/elasticsearch-java/releases/tag/v8.7.0

Reply:
Good point. I just read the elasticsearch java client is [forward compatible](https://www.elastic.co/guide/en/elasticsearch/client/java-api-client/current/introduction.html#:~:text=Elasticsearch%20language%20clients%20are%20only,distributions%20and%20without%20guarantees%20made.) while the HLRC was just backward compatible.

Regarding the ingester, I am not sure it's already at the same level as the HLRC BulkProcessor. Retries seem to be missing. I would prefer to have them implemented in the client instead of implementing this logic in OAK.

Reply:
I would go ahead with the bulk ingester when this gets fixed https://github.com/elastic/elasticsearch-java/issues/478